policyengine_us/variables/gov/states/ia/tax/income/net_income/ia_pension_exclusion_eligible.py

```diff
@@ -19,7 +19,6 @@ def formula(person, period, parameters):
         # ... determine age eligibility
         age = person("age", period)
         age_eligible = age >= p.minimum_age
-        head_or_spouse_age_eligible = is_head_or_spouse & age_eligible
```

Comment:
I think we need to use this instead of age_eligible on line 25. From the form:
>If **you or your spouse** receive a pension, an annuity, a self-employed retirement plan, deferred compensation,
IRA distribution, or other retirement plan benefits, you may be eligible to exclude from Iowa income tax part
or all of the retirement income that is taxable on your federal return. The Roth conversion income, included in
net income, is eligible for this exclusion.